### PR TITLE
Add AppHub support

### PR DESCRIPTION
**Pull request by orta**
I did some research, [AppHub](https://deploy.apphub.io) looks like the simplest option WRT the hosted JS servers. Runs entirely in Emission's Example App, which is a really big 👍 

API seems easy to handle WRT to the continuous deployment - I'll need to put some work in to get the Enterprise beta deployment working. Think we'll have to have it given how often the native side is being updated. Will try get travis doing it, like we have circle deploying the other apps. @ashfurrow is eidolon auto-deployed via circle etc?

I'll start looking into docs etc tomorrow. I've sent invites out to people.

### CODE REVIEW

**Comment by ashfurrow** [comment]
Eidolon is deployed manually, but it's a one-step command to deploy. The deploy also adds a commit that updates the changelog and some plists.

**Comment by orta** [comment]
Cool, thanks @ashfurrow 

@alloy - because this is done entirely in the host app, we also have the option of doing the same thing inside eigen too

**Comment by alloy** [comment]
Nice 👌

**Comment by orta** [comment]
OK, this is now auto-deploying on master

**Comment by orta** [comment]
Also doing some work on the tools: https://github.com/joshuapinter/apphubdeploy/pull/5

**Comment by orta** [comment]
I've also submitted a PR to reduce the amount of React deps that AppHub brings in - https://github.com/AppHubPlatform/apphub/pull/32

**Comment by orta** [comment]
Alright, this is good to go @mennenia - I'm assigning it to you as @alloy is in 🗾

**Comment by orta** [comment]
You can see each build we ship here: https://dashboard.apphub.io/projects/Z6IwqK52JBXrKLI4kpvJ

screen shot 2016-09-05 at 3 45 43 pm

**Comment by KrauseFx** [comment]
Looking at their website, I was surprised to see SoundCloud on there, I completely missed their blog post https://developers.soundcloud.com/blog/react-native-at-soundcloud

**Comment by orta** [comment]
Totally, I _really love_ this part too:

> Meanwhile and independently, the design team was running user-testing sessions with React Native-based prototypes and we also had capacity from very eager web engineers

File: README.md [hunk 1]
```diff
@@ -2,8 +2,16 @@
...
 [React Native] Components used by [Eigen].
 
-[React Native]: http://facebook.github.io/react-native/
-[Eigen]: https://github.com/artsy/eigen
+### Meta
+
+* __State:__ production
+* __Point People:__ [@alloy](https://github.com/alloy)
+
+This is a core [Artsy Mobile](https://github.com/artsy/mobile) OSS project, along with [Energy](https://github.com/artsy/energy), [Eidolon](https://github.com/artsy/eidolon), [Eigen](https://github.com/artsy/eigen) and [Emergence](https://github.com/artsy/emergence).
```
Comment (mennenia) [inline README.md]: Nicely snuck in :D

**Comment by mennenia** [comment]
Looks great Orta! Very excited to use this.

**Pull request merged**